src/bokeh/sphinxext/_internal.py

```diff
@@ -0,0 +1,17 @@
+# -----------------------------------------------------------------------------
+# Copyright (c) Anaconda, Inc., and Bokeh Contributors.
+# All rights reserved.
+#
+# The full license is in the file LICENSE.txt, distributed with this software.
+# -----------------------------------------------------------------------------
+
+import os
+import pathlib
+
+__all__ = (
+    "_REPO_TOP",
+)
+
+# TODO (bv) this needs to be a configuration
+# issue #14499; moved from bokeh.sphinxext.util
```

Comment:
The comment can be removed. This actually does not need to be configurable since it only applies to (soon to be) internal extensions. Also would prefer renaming to `REPO_TOP` (no underscore) Since the attribute is in a private module, it does not need to be private, itself.